src/main/java/mate/academy/spring/model/MovieSession.java

```diff
@@ -1,61 +1,34 @@
 package mate.academy.spring.model;
 
 import java.time.LocalDateTime;
+import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.JoinColumn;
 import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 
+@Data
 @Entity
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "movie_sessions")
 public class MovieSession {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     @ManyToOne
     private Movie movie;
     @ManyToOne
+    @JoinColumn(name = "cinema_hall")
     private CinemaHall cinemaHall;
+    @Column(name = "show_time")
```

Comment:
`JFYI` under the hood hibernate will call it the same, even without attribute `name`